src/ragger/conftest/base_conftest.py

```diff
@@ -224,6 +226,9 @@ def use_only_on_backend(request, backend_name):
         if current_backend != backend_name:
             pytest.skip(f'skipped on this backend: "{current_backend}"')
 
+@pytest.fixture(autouse="function")
+def default_screenshot_path(request):
+    return request.path.parent.resolve()
```

Comment:
Not sure how this one behaves depending on where pytest is launched?
Maybe something based on a ROOT_SCREENSHOT_PATH to be defined in conftest.py or something could be better?